dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -610,13 +610,27 @@ private Workflow createOrGetWorkflow(Class workflowType, String repository, User
             }
         }
 
+        if (workflowType == BioWorkflow.class || workflowType == AppTool.class) {
```

Comment:
What is left? Is it services, if so maybe they shouldn't be allowed to change languages either.